packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -35,7 +35,7 @@ import { OpKind, ParamsWithKind, TransferTicketParams } from '../../src/operatio
 import { NoopParser } from '../../src/taquito';
 import { OperationBatch } from '../../src/batch/rpc-batch-provider';
 import { ContractMethodObject } from '../../src/contract/contract-methods/contract-method-object-param';
-import { ticketTokenTestMock } from '../helpers';
+import { anotherMapTypecheckError, smallerTypeCheckError, ticketTokenTestMock } from '../helpers';
```

Comment:
same here with the naming, `anotherMapTypecheckError` doesn't sound very descriptive

Reply:
it's also not used anywhere it seems, it's failing the linter